Libs/WorkflowsCommonSteps/src/main/java/rapture/dp/invocable/configuration/steps/ConfigurationStep.java

```diff
@@ -74,10 +75,9 @@ public String invoke(CallingContext ctx) {
             if (lio < 0) lio = 0;
 
             StringBuilder externalUrl = new StringBuilder();
-            String host = System.getenv("HOST");
-            String port = System.getenv("PORT");
-            externalUrl.append("http://").append((host != null) ? host : LOCALHOST).append(":").append((port != null) ? port : DEFAULT_RIM_PORT)
-                    .append("/process/")
+            String host = getUIHostName();
+            String port = getUIPort();
+            externalUrl.append("http://").append(host).append(":").append(port).append("/process/")
                     .append(docPath.substring(0, lio)).append(WORKORDER_DELIMETER).append(docPath.substring(lio + 1));
             decision.setContextLiteral(ctx, workOrderUri, EXTERNAL_RIM_WORKORDER_URL, externalUrl.toString());
```

Comment:
I know the references to 'RIM' are legacy; but i'd rather if more generic terms are used. For example, EXTERNAL_UI_WORKORDER_URL instead of EXTERNAL_RIM_WORKORDER_URL